metadata.rb

```diff
@@ -1,13 +1,11 @@
-name 'wix'
-maintainer 'Chef Software, Inc.'
-maintainer_email 'cookbooks@chef.io'
-license 'Apache-2.0'
-description 'Installs/Configures Windows Installer XML toolset (WiX)'
-version '5.0.0'
-supports 'windows'
-
-depends 'windows', '>= 2.0'
+name              'wix'
+maintainer        'Sous Chefs'
+maintainer_email  'help@sous-chefs.org'
+license           'Apache-2.0'
+description       'Installs/Configures Windows Installer XML toolset (WiX)'
+version           '5.0.0'
+source_url        'https://github.com/sous-chefs/wix'
+issues_url        'https://github.com/sous-chefs/wix/issues'
+chef_version      '>= 13.4'
```

Comment:
This needs to be updated since you're using the `archive_file` resource which doesn't introduced into Chef 15.
```suggestion
issues_url        'https://github.com/sous-chefs/wix/issues'
chef_version      '>= 15.0'
```